docs/Reference/Warnings.md

```diff
@@ -0,0 +1,36 @@
+
+<h1 align="center">Fastify</h1>
+
+## Warnings
+
+### Warnings In Fastify
+Warnings are enabled by default, and you can disable it by:
+
+- setting the `NODE_NO_WARNINGS` environment variable to `1`
+- passing the `--no-warnings` flag to the node process
+- setting 'no-warnings' in the `NODE_OPTIONS` environment variable
+
+However, disabling warnings is not recommended as it may cause
+potential problems when upgrading Fastify versions.
+Only experienced users should consider disabling warnings.
+
+### Fastify Warning Codes
+
+| Code | Description | How to solve |
+| ---- | ----------- | ------------ |
+| **FSTWRN001** | The specified schema for a route is missing. This may indicate the schema is not well specified. | Check the schema for the route. |
+| **FSTDEP005** | You are accessing the deprecated `request.connection` property. | Use `request.socket`. |
+| **FSTDEP006** | You are decorating Request/Reply with a reference type. This reference is shared amongst all requests. | Use `onRequest` hook. |
+| **FSTDEP007** | You are trying to set a HEAD route using `exposeHeadRoute` route flag when a sibling route is already set. | Use `exposeHeadRoutes` from the server instance. |
```

Comment:
> Use `exposeHeadRoutes` from the server instance.

Should be:
Remove `exposeHeadRoutes` or explicitly set `exposeHeadRoutes` to `false`